activerecord/lib/active_record/railties/databases.rake

```diff
@@ -382,11 +382,14 @@ db_namespace = namespace :db do
         base_dir
       end
 
+      file_fixtures_dir = File.join(fixtures_dir, "files")
```

Comment:
this could be inlined on line 391. it's only needed in one branch of the if/else.

Reply:
@ghiculescu addressed in https://github.com/rails/rails/pull/42153/commits/ac0a257dbe661d94de3c6475187e3fb6c0a1264d.